cantools/database/can/signal.py

```diff
@@ -339,7 +339,7 @@ def choice_string_to_number(self, string: str) -> int:
 
         raise KeyError(f"Choice {string} not found in Signal {self.name}.")
 
-    def get_offset_scaling(self, val: float = 0.0) -> Tuple[float, float]:
+    def get_offset_scaling(self, raw_val: Optional[float] = None, scaled: Optional[float] = None) -> Tuple[float, float]:
```

Comment:
Ah I didnt know that the `Signal` class is also used for arxml signals, in that case yes it should support it and I'll properly add it to that class as well.

However, since I have not really worked with arxml files that much so far I would not feel very comfortable editing its parsing to make it fully work. But maybe someone can then link it up once my part is done :)

Reply:
>  However, since I have not really worked with arxml files that much so far I would not feel very comfortable editing its parsing to make it fully work. But maybe someone can then link it up once my part is done :)

don't worry, I'll take over for the ARXML stuff. eventually ;)